native/c/examples/timer/testtime.c

```diff
@@ -0,0 +1,54 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+#include "ihasdwa.h"
+#include "zmetal.h"
+#include "zrecovery.h"
+#include "zwto.h"
+#include "zecb.h"
+
+#pragma prolog(SOMEFUNC, " ZWEPROLG NEWDSA=(YES,8),SAVE=BAKR,SAM64=YES")
+#pragma epilog(SOMEFUNC, " ZWEEPILG ")
+void SOMEFUNC(void *PTR32 parameter)
+{
+  zwto_debug("@TEST called some func");
```

Comment:
I do have a question offline about the use of `zwto_debug`, I'm probably messing something up but I'm running into an SD23 abend when trying to use it in #445 😓 

It seems super helpful for debugging in Metal C, and it looks like you may have been using it already for testing these changes

Reply:
My guess is the metal c stack is not initialized... you'll probably need a `pragma prolog(xxxxxxxx, "ZWEPROLG NEWDSA=(YES,64)"` and corresponding `#pragma epilog(xxxxxxxx, "ZWEEPILG")`